plugins/woocommerce/src/Blocks/BlockTypes/Cart.php

```diff
@@ -246,8 +247,8 @@ protected function enqueue_data( array $attributes = [] ) {
 		$this->asset_data_registry->add( 'isBlockTheme', wc_current_theme_is_fse_theme(), true );
 		$this->asset_data_registry->add( 'activeShippingZones', CartCheckoutUtils::get_shipping_zones(), true );
 
-		$pickup_location_settings = get_option( 'woocommerce_pickup_location_settings', [] );
-		$this->asset_data_registry->add( 'localPickupEnabled', wc_string_to_bool( $pickup_location_settings['enabled'] ?? 'no' ), true );
+		$pickup_location_settings = LocalPickupUtils::get_local_pickup_settings();
+		$this->asset_data_registry->add( 'localPickupEnabled', wc_string_to_bool( $pickup_location_settings['enabled'] ), true );
```

Comment:
Nitpic: What are your thoughts if we return `$pickup_location_settings['enabled']` as a boolean from `LocalPickupUtils::get_local_pickup_settings();`? That way, we don't need `wc_string_to_bool` to convert the string to a boolean. I'm thinking about something like this before we return `$pickup_location_settings`:

```php
if ( isset( $pickup_location_settings['enabled'] ) ) {
    $pickup_location_settings['enabled'] = wc_string_to_bool( $pickup_location_settings['enabled'] );
}
```

Reply:
Good idea, added! Thanks